src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSWASubmissionStatus.tsx

```diff
@@ -0,0 +1,246 @@
+import ChevronRightIcon from "@artsy/icons/ChevronRightIcon"
```

Comment:
This is the file containing the new submission status section on the MyC artwork page.